concourse/scripts/pipelines-bosh-cloudfoundry.sh

```diff
@@ -18,6 +19,9 @@ cf_grafana_version=$("${SCRIPT_DIR}"/val_from_yaml.rb releases.grafana.version "
 generate_vars_file() {
    cat <<EOF
 ---
+pipeline_name: ${pipeline_name}
+makefile_env_target: ${MAKEFILE_ENV_TARGET:-dev}
+self_update_pipeline: ${SELF_UPDATE_PIPELINE:-false}
```

Comment:
The script itself prints how to disable the feature, so I will not put it in the README.